pkg/k8s/services/crud.go

```diff
@@ -48,10 +48,31 @@ func Deploy(ctx context.Context, s *apiv1.Service, c kubernetes.Interface) error
 		oktetoLog.Infof("created service '%s'", s.Name)
 	} else {
 		oktetoLog.Infof("updating service '%s'", s.Name)
-		old.Annotations = s.Annotations
-		old.Labels = s.Labels
+
+		for k, v := range s.Annotations {
```

Comment:
Adding annotations and/or labels will break compose deploy logic. For example, if I drop labels/annotations in my docker compose yaml file, `okteto deploy` will keep the previous ones.

Reply:
For this fix, I wouldn't mind if we lose the divert annotations, and we do a proper fix later, for example, by running the divert logic after the docker compose logic